Update quick-start.md

Barnoid- opened the pull request:
- version bump to 2.5
- fix broken link to ember-cli-deploy
- add info on how to deploy to an Apache server
  (there are lots of hits in Google about this and many misleading/nonworking suggestions. The simplest working one seems to be FallbackResource)

locks commented on source/localizable/getting-started/quick-start.md:
````diff
@@ -15,7 +15,7 @@ You can install Ember with a single command using npm, the Node.js package
 manager. Type this into your terminal:
 
 ```sh
-npm install -g ember-cli@2.4
````
Please remove this line from the PR

locks commented on source/localizable/getting-started/quick-start.md:
```diff
@@ -256,4 +256,12 @@ everyone relies on. If you're interested in deploying your app to
-[ember-deploy]: http://ember-cli.github.io/ember-cli-deploy/
+[ember-deploy]: http://ember-cli-deploy.github.io/ember-cli-deploy/
+
+If you deploy your application to an Apache web server, first create a 
```
Can you insert a newline after each comma as per our [style guide](https://github.com/emberjs/guides/blob/master/CONTRIBUTING.md#style-guide)? Thanks :)

locks commented on source/localizable/getting-started/quick-start.md:
```diff
@@ -256,4 +256,12 @@ everyone relies on. If you're interested in deploying your app to
 Deploy][ember-deploy] addon.
 
-[ember-deploy]: http://ember-cli.github.io/ember-cli-deploy/
+[ember-deploy]: http://ember-cli-deploy.github.io/ember-cli-deploy/
```
Can you send this fix as a separate PR? makes it easier for us to review+merge.

Barnoid- commented:
- removed version bump as instructed
- removed URL fix for Ember CLI Deploy (will create separate PR)
- this leaves only the instructions for deployments on Apache

Barnoid- commented:
@locks: btw, I'm not sure where you wanted a comma. Let me know, I'll try to be consistent afterwards.

homu commented:
:umbrella: The latest upstream changes (presumably #1395) made this pull request unmergeable. Please resolve the merge conflicts.

locks commented:
@Barnoid- I apologise, I mixed up my terms :P I meant that you should insert line breaks after each period, the suggestion that you mentioned.
Most of the guides don't follow the style guide because they are older documents and we don't have the bandwidth to go through them and change, so we'll adhere to it for any new content that we write.
The document itself is just for contributors/maintainers, so it doesn't need to follow suit.
Will merge after the conflict is resolved, appreciated!

Barnoid- commented:
@locks Okay, got it. Will do the rewrite as soon as I figure out how.
(avid SVN user and still feeling quite lost here. Can I resolve using the github site only? Couldn't figure out how to do that, and, embarrassingly, not even how to clone the patch to my local drive. All I got was the master from when I created the patch...sigh. If you can spare a minute and point me in the right direction I'd certainly appreciate it.)

locks commented:
@Barnoid- are you in the Ember Slack channel?

locks commented:
🎉  welcome to the hellscape that is git 🎉 
(at least it's a working hellscape)
@homu r+

homu commented:
:pushpin: Commit 67d1cef has been approved by `locks`

<!-- @homu r=locks 67d1cef8738b0a20ea04ca9ccdb520483ee872df -->

homu commented:
:hourglass: Testing commit 67d1cef with merge 78b42ca...

homu commented:
:sunny: Test successful - [status](https://travis-ci.org/emberjs/guides/builds/125369111)
